.eslintrc.js

```diff
@@ -761,6 +761,7 @@ const extendsList = [
 if (detectDeprecations) {
   extendsList.push('plugin:deprecation/recommended');
 }
+extendsList.push('prettier'); // Keep this last in the list
```

Comment:
can we put a comment that this is eslint-config-prettier its hard to know what package is being used

Reply:
```suggestion
extendsList.push('prettier'); // From package: `eslint-config-prettier`. Keep this last in the list
```